go.mod

```diff
@@ -108,3 +108,5 @@ require (
 )
 
 replace sigs.k8s.io/apiserver-network-proxy/konnectivity-client => sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.27
+
+replace open-cluster-management.io/addon-framework => github.com/yue9944882/addon-framework v0.0.0-20220315104416-d9192aa14201
```

Comment:
I guess it should depend on the addon-framework repo under ocm.

Reply:
updated to latest commit of addon-framework